### PR TITLE
Add Movement(Beginner) trick to Hall of Combat Mastery

### PR DESCRIPTION
**Pull request by f-raZ0R**
After breaking the HoCM tube, you can NSJ Screw to get enough distance to leave the portal.

### CODE REVIEW

**Comment by f-raZ0R** [comment]
> Add an entry in the changelog.
> 
> Can you add a video for this?

Sure thing, does this look good?
Wasn't really sure what format you wanted the changelog addition to be in, as I couldn't find any documentation on it in the repo, so I just added onto it based on what was already there.

**Comment by dyceron** [comment]
> > Add an entry in the changelog.
> > Can you add a video for this?
> 
> Sure thing, does this look good? Wasn't really sure what format you wanted the changelog addition to be in, as I couldn't find any documentation on it in the repo, so I just added onto it based on what was already there.

Streamable isn't really a reliable site for uploads since overtime they get deleted, unless you pay to have videos last longer.

**Comment by f-raZ0R** [comment]
> 
> Streamable isn't really a reliable site for uploads since overtime they get deleted, unless you pay to have videos last longer.

What alternatives would you recommend?

**Comment by duncathan** [comment]
YouTube.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
...
+- Nothing.
+
+### Metroid Prime 2: Echoes
+
+- Nothing.
+
+#### Logic Database
+
+- Added: Movement(Beginner) trick for crossing HoCM from the Portal Side, after the tunnel is destroyed, with NSJ Screw Attack. 
+
```
Comment (XenoWars) [inline CHANGELOG.md:18]: The entry should be moved under to the existing Echoes section that was added with Inverted Aether

**Review by XenoWars — CHANGES_REQUESTED** [review]

**Review by henriquegemignani — APPROVED** [review]

**Pull request merged**